### PR TITLE
DO NOT MERGE [New project structure] [Ecr-2867] Fixed links to the new repo name

### PR DESCRIPTION
**Pull request by bullet-tooth**
## Overview

---
See: https://jira.bf.local/browse/ECR-2867

Should be double checked and merged after the repo renaming.

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Exonum Java Libraries
 
-[![Build Status](https://img.shields.io/travis/exonum/exonum-java-binding/master.svg)](https://travis-ci.com/exonum/exonum-java-binding)
-[![Coverage Status](https://coveralls.io/repos/github/exonum/exonum-java-binding/badge.svg?branch=master)](https://coveralls.io/github/exonum/exonum-java-binding?branch=master)
+[![Build Status](https://img.shields.io/travis/exonum/exonum-java/master.svg)](https://travis-ci.com/exonum/exonum-java)
```
Comment (bullet-tooth) [inline README.md]: should be additionally checked after renaming

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+[![Coverage Status](https://coveralls.io/repos/github/exonum/exonum-java/badge.svg?branch=master)](https://coveralls.io/github/exonum/exonum-java?branch=master)
 [![Join the chat on https://gitter.im/exonum/exonum-java-binding](https://img.shields.io/gitter/room/exonum/exonum-java-binding.svg?label=Chat)](https://gitter.im/exonum/exonum-java-binding)
```
Comment (bullet-tooth) [inline README.md]: do we need to rename Gitter channel? 🤔

Comment (dmitry-timofeev) [reply README.md]: Hardly anyone uses it — Java questions get asked in the main ones (Exonum and ruExonum)

**Comment by bullet-tooth** [comment]
Replaced by #696

**Pull request closed without merging**